modules/message-lane/src/benchmarking.rs

```diff
@@ -81,6 +81,10 @@ pub struct MessageDeliveryProofParams<ThisChainAccountId> {
 
 /// Trait that must be implemented by runtime.
 pub trait Config<I: Instance>: crate::Config<I> {
+	/// Lane id to use in benchmarks.
+	fn bench_lane_id() -> LaneId {
+		Default::default()
+	}
```

Comment:
Why did this need to get moved into the `Config` trait? It looked fine as a standalone function

Reply:
We're blocking all lanes except default in our runtimes + will be blocking all lines except *some-lane* in P+K. So it's controlled by runtime